cmd/cluster/resize/infra_node.go

```diff
@@ -536,3 +543,30 @@ func (r *Resize) nodesMatchExpectedCount(ctx context.Context, labelSelector labe
 
 	return false, nil
 }
+
+// having an error when beeing in a rety loop, should not be handled as an error, and we should just disaply it and continue
+// in case we have a function that return a bool statis and an error, we can use following helper
+// f beeing a function returning (bool, error), replace
+//
+//	return f(...)
+//
+// by
+//
+//	return skipError(wrapResult(f(...)), "message to context the error"
+//
+// and then the return will always have error set to nil, but a continuing message will be dispalyed in case of error
```

Comment:
```suggestion
// having an error when being in a rety loop, should not be handled as an error, and we should just display it and continue
// in case we have a function that return a bool status and an error, we can use following helper
// f being a function returning (bool, error), replace
//
//	return f(...)
//
// by
//
//	return skipError(wrapResult(f(...)), "message to context the error"
//
// and then the return will always have error set to nil, but a continuing message will be dispalyed in case of error
```